### ISSUE TITLE
Make possible to add params on URI and by queryParams simultaneously for Http Requests

### ISSUE THREAD

**Issue by kabarret** [authorjapps/zerocode#383]
As software engineer I would like to be possible have query parameter on my URI and on queryParams block.

```{
            "name": "find_combing_url_and_query_params",
            "url": "/api/v1/search/persons?lang=Amazing",
            "operation": "GET",
            "request": {
                "queryParams": {
                    "city": "Lon"
                }
            },
            "assertions": {
                "status": 200,
                "body" : {
                    "exactMatches" : true,
                    "name" : "Mr Bean",
                    "lang" : "Amazing",
                    "city" : "Lon"
                }
            }
        }
``` 

Today it's no possible, and what happen is an error of bad format url. 
like :
`/api/v1/search/persons?lang=Amazing?city=Lon`

**Comment by kabarret** [comment]
I just code a solution for this, and I gonna open a PR, is little similar at was discussed in [https://github.com/authorjapps/zerocode/issues/148](url).
I made some refactor and use Apache URI Builder to handle with that, and others possibles situations

**Comment by authorjapps** [comment]
To be available in `1.3.19`
